resources/dicts/relationship_events/normal_interactions/dislike/decrease.json

```diff
@@ -1,109 +1,151 @@
+			"m_c and r_c spend some time together and nearly forget what made them upset with one another in the first place.",
+			"m_c and r_c both apologize to each other at the same time.",
+			"When explaining something, m_c is surprised that r_c understands.",
+			"m_c hesitantly asks to exchange prey with r_c. r_c agrees, since m_c's prey is {PRONOUN/r_c/poss} favorite kind, anyways.",
+			"The Clan is surprised to see m_c and r_c casually chatting over a meal.",
+			"m_c cheers r_c up after a small disagreement."
+		],
+		"reaction_random_cat": {
+			"dislike": "decrease"
+		}
+	},
+	{
+		"id": "dislike_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c is surprised to hear r_c express an ideal {PRONOUN/m_c/subject} can agree with.",
+			"m_c didn't realize how generous r_c could be.",
+			"m_c sees r_c stand up for a Clanmate despite the pushback from their peers.",
+			"m_c overhears r_c saying something kind about {PRONOUN/m_c/object}.",
+			"m_c tries to look at r_c's actions from another perspective, and starts to like {PRONOUN/r_c/subject} more.",
+			"m_c notices that some of {PRONOUN/m_c/poss} Clanmates talk very highly of r_c.",
+			"The way r_c jumps to help a Clanmate makes m_c think {PRONOUN/r_c/subject} might not be all that bad.",
+			"There's something soothing about r_c that m_c never realized before.",
+			"m_c realizes how smart r_c is when it comes to certain things.",
+			"m_c talks to one of r_c's friends and begins to understand why others like {PRONOUN/r_c/object}.",
+			"m_c considers how to make amends with r_c after a long night of deliberation."
+		]
+	},
+	{
+		"id": "dislike_de_high2",
+		"intensity": "high",
+		"interactions": [
+			"m_c and r_c joke around about how bad the other Clans smell.",
+			"r_c can't help but laugh when m_c tells a funny joke.",
+			"A Clanmate claims that m_c and r_c were sharing tongues.",
+			"m_c and r_c sincerely apologize to one another.",
+			"On a cold night, m_c decides to share a nest with r_c.",
+			"On a sunny day, m_c and r_c happen to sun bathe in the same place. They enjoy the silence.",
+			"r_c admits that m_c did something clever.",
+			"r_c sacrifices treatment for a cough in favor of m_c's ailment being addressed.",
+			"m_c and r_c fight over a piece of prey, but soon decide to share it instead.",
+			"The Clan has noticed m_c and r_c are smiling at each other more today.",
+			"m_c and r_c realize they have more in common than previously thought.",
+			"r_c thinks that m_c isn't so bad after all."
+		],
+		"reaction_random_cat": {
+			"dislike": "decrease"
+		}
+	},
+	{
+		"id": "dislike_de_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c and r_c heckled another Clan at the gathering together.",
+			"m_c and r_c worked well together on a patrol.",
+			"When they return from a patrol, m_c and r_c seem more at peace with each other than usual.",
+			"After a patrol gone wrong, m_c helps an injured r_c back to camp.",

```

Comment:
Would have to remove this, as we can't guarantee it happens after a cat is injured.